node/service/src/chain_spec/vara.rs

```diff
@@ -365,15 +461,40 @@ fn testnet_genesis(
                 .map(|x| (x.0.clone(), x.1.clone(), STASH, StakerStatus::Validator))
                 .collect(),
             invulnerables: initial_authorities.iter().map(|x| x.0.clone()).collect(),
-            force_era: pallet_staking::Forcing::ForceNone,
             slash_reward_fraction: Perbill::from_percent(10),
             ..Default::default()
         },
+        democracy: DemocracyConfig::default(),
+        elections: ElectionsConfig {
+            members: endowed_accounts
+                .iter()
+                .take((num_endowed_accounts + 1) / 2)
+                .cloned()
+                .map(|member| (member, STASH))
+                .collect(),
```

Comment:
Not sure it will be faster but using slice is a little bit shorter.
```suggestion
            members: endowed_accounts[0..(num_endowed_accounts + 1) / 2]
                .iter()
                .map(|member| (member.clone(), STASH))
                .collect(),
```

Reply:
I just borrowed this piece from Substrate as is - haven't really looked into optimising it. Performance doesn't matter here at all considering it's a one-time action at start-up, and the number of items in the collection is small. But having 2 less lines in the code is a good reason to accept this suggestion.